Trakt/Api/ServerApiEndpoints.cs

```diff
@@ -126,47 +105,67 @@ public TraktUriService(TraktApi traktApi, ILogger logger, ILibraryManager librar
         }
 
 
+        public object Post(DeviceAuthorization deviceAuthorizationRequest)
+        {
+            _logger.LogInformation("DeviceAuthorization request received");
 
+            // Create a user if we don't have one yet - TODO there should be an endpoint for this that creates a default user
```

Comment:
This could be done in a separate PR. It's a bit of an issue that the frontend shows user defaults without having created a config yet imo. The frontend should call the backend to create a default user if one does not exist.

Reply:
Why do we even want a plugin to create a user? Or am I misunderstanding something?

Reply:
Bad wording. It creates a config for an existing Jellyfin user.